### ISSUE TITLE
Decouple tutorial from vercel sql functionality

### ISSUE THREAD

**Issue by openSourceBugs** [vercel/next-learn#619]
This is completely unnecessary and pretty much breaks the entire tutorial for any other tools.

**Comment by Gastonite** [comment]
I agree it's a pain in the ass to have to find an alternative to make the tutorial working with a locally installed postgres database ...

**Comment by jon23d-coalition** [comment]
I'd really like to do this tutorial, but without having to link a github account to a third-part service. I have postgres locally, but supplying a connection string in the .env doesn't seem to be enough. This shouldn't be so hard.

**Comment by sbarre** [comment]
I'll second what others have said here.

I respect that tutorials can be funnels into your products, nothing wrong with getting a return on your investment, but at the very least you should lay out the tutorial requirements **at the start**, so it's clear _before_ someone starts that they will need GitHub/Vercel accounts part way through, and then they can make an informed decision about proceeding or not..

After going through the first 5 chapters and enjoying myself, my tutorial experience ground to a disappointing halt when it went from doing all the work locally to suddenly having to create an account and pushing code and switching to your cloud services.

**Comment by leerob** [comment]
https://github.com/vercel/next-learn/pull/989
